libraries/resource_providers/api1600/synergy/logical_interconnect_provider.rb

```diff
@@ -0,0 +1,20 @@
+# (c) Copyright 2020 Hewlett Packard Enterprise Development LP
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed
+# under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+# CONDITIONS OF ANY KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations under the License.
+
+module OneviewCookbook
+  module API1600
+    module Synergy
+      # LogicalInterconnect API1600 Synergy provider
+      class LogicalInterconnectProvider < API1200::C7000::LogicalInterconnectProvider
```

Comment:
try to inherit it from 1200 synergy module. Somehow API1200 synergy is inherited from C7000 but lets correct it in API1600.